api/src/opentrons/config/advanced_settings.py

```diff
@@ -460,6 +452,16 @@ def _migrate14to15(previous: SettingsMap) -> SettingsMap:
     return newmap
 
 
+def _migrate15to16(previous: SettingsMap) -> SettingsMap:
+    """Migrate to version 16 of the feature flags file.
+
+    - Removes deprecated enableProtocolEngine option
```

Comment:
Looks like a copy-paste mistake?

```suggestion
    - Removes deprecated enableHeaterShakerPAPI option
```